matter/wifi/wf200/efr_spi.c

```diff
@@ -303,18 +335,36 @@ void sl_wfx_host_start_platform_interrupt(void)
   GPIO_ExtIntConfig(SL_WFX_HOST_PINOUT_SPI_WIRQ_PORT, SL_WFX_HOST_PINOUT_SPI_WIRQ_PIN, wirq_irq_nb, true, false, true);
 }
 
-/****************************************************************************
- * Disable WFX interrupt
- *****************************************************************************/
-sl_status_t sl_wfx_host_disable_platform_interrupt(void)
+sl_status_t sl_wfx_host_enable_platform_interrupt(void)
 {
-  GPIO_IntDisable(1 << wirq_irq_nb);
+  GPIO_IntEnable(1 << wirq_irq_nb);
   return SL_STATUS_OK;
 }
+#elif defined(EFR32MG24)
+void sl_wfx_host_start_platform_interrupt(void)
+{
```

Comment:
Why separate both? You are creating a lot of confusion and duplicating the work.

#if defined(EFR32MG12)
`sl_wfx_host_start_platform_interrupt`

#elif defined(EFR32MG24)
`sl_wfx_host_start_platform_interrupt`


yet both function do exactly the same with exactly the same parameters

```
  // Enable (and clear) the bus interrupt
  GPIO_ExtIntConfig(SL_WFX_HOST_PINOUT_SPI_WIRQ_PORT, SL_WFX_HOST_PINOUT_SPI_WIRQ_PIN, wirq_irq_nb, true, false, true);
```

Reply:
Maintained same code for both MG12 and MG24.